docs/lokalise2_branch_create.md

```diff
@@ -29,4 +29,4 @@ lokalise2 branch create [flags]
 
 * [lokalise2 branch](lokalise2_branch.md)	 - Manage branches
 
-###### Auto generated by spf13/cobra on 23-Nov-2020
+###### Auto generated by spf13/cobra on 21-Dec-2020
```

Comment:
Should we discard this kind of date bump for pages which weren't actually updated? It's impossible to review such commits

Reply:
I'm as well for that